package.json

```diff
@@ -262,7 +262,10 @@
       "eslint --fix",
       "git add"
     ],
-    "**/*.{js,json}": "yarn prettier:write"
+    "**/*.{js,json}": [
+      "yarn prettier:write",
+      "git add"
```

Comment:
What's the reason for this?

Reply:
Sometimes when committing I get modified JSON files afterwards that haven't been included in the commit itself. I assumed this was the reason.